aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/service/StorageService.java

```diff
@@ -127,9 +131,16 @@ TransferObserver uploadInputStream(@NonNull String transferId,
      * @param prefix path appended to S3 keys
      * @param pageSize number of keys to be retrieved from s3
      * @param nextToken next continuation token to be passed to s3
+     * @param subpathStrategy SubpathStrategy to include/exclude sub-paths
      * @return A list of parsed items present inside given path
      */
-    StorageListResult listFiles(@NonNull String path, @NonNull String prefix, int pageSize, @Nullable String nextToken);
+    StorageListResult listFiles(
```

Comment:
This should be a new function - we can't add parameters to this function since it's public.

Reply:
Updated to new function